### PR TITLE
enhancer: removed country from records

### PR DESCRIPTION
**Pull request by ErnestaP**
* Removed country from records with affiliation value with: cooperation agreement with cern.
* ref: https://github.com/cern-sis/issues-scoap3/issues/176

### CODE REVIEW

File: dags/common/enhancer.py [hunk 1]
```diff
@@ -42,6 +43,18 @@ def __construct_titles(self, item, publisher):
             }
         ]
 
+    def __remove_country(self, item):
+        for author in item.get("authors", []):
+            for affiliation in author.get("affiliations", []):
+                pattern_for_cern_cooperation_agreement = re.compile(
+                    r"cooperation agreement with cern", re.IGNORECASE
```
Comment (drjova) [inline dags/common/enhancer.py]: if we have the `compile` inside the loops it's loosing its advantage over the simple `match`, especially in a double for loop.

**Review by drjova — CHANGES_REQUESTED** [review]
Thank you 😃 , there is a comment that we have to address.

File: tests/units/common/test_enhancer.py [hunk 2]
```diff
@@ -60,6 +60,128 @@
...
+    "subtitle": "subtitle",
+}
+expected_output_with_affiliation_of_cooperation_agreement_with_CERN = {
+    "abstracts": [{"value": "this is abstracts", "source": publisher}],
+    "authors": [
+        {
+            "affiliations": [
+                {
+                    "organization": "International laboratory covered by a cooperation agreement with CERN",
+                    "value": "International laboratory covered by a cooperation agreement with CERN, Geneva, Switzerland",
```
Comment (drjova) [inline tests/units/common/test_enhancer.py:121]: we are not testing the case that `country` already populated from the XML files.

File: dags/common/enhancer.py (hunk 1 above)
Comment (drjova) [reply dags/common/enhancer.py]: @ErnestaP ☝️

**Review by drjova — APPROVED** [review]

**Comment by drjova** [comment]
@ErnestaP could you please squash?

**Comment by drjova** [comment]
@ErnestaP why there is a merge commit?

**Comment by ErnestaP** [comment]
Merge commit because of resolving the merge conflict by using UI

**Pull request merged**